package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "axios",
-  "version": "0.18.0",
+  "version": "0.19.0",
```

Comment:
Please don't update the version, this gets done automatically when a release is made.

Reply:
done